solr/core/src/java/org/apache/solr/metrics/reporters/SolrSlf4jReporter.java

```diff
@@ -85,7 +85,7 @@ public void report() {
     }
 
     @Override
-    @SuppressWarnings({"rawtypes"})
+    @SuppressWarnings("rawtypes") // super uses raw Gauge, so we're stuck with it
```

Comment:
> // super uses raw Gauge, so we're stuck with it

Thanks for adding the comment here!

I haven't yet looked but in principle would be curious if the super class could potentially (in a future version) remove the raw use, in which case we could in the comment also reference the JIRA or other issue corresponding to such a change.

Reply:
I just checked and it is fixed in the development branch of DropWizard 5.0, but not in the stable branch for 4.x. It looks like 5.0 is on pause though because they are concerned that there wouldn't be adoption of incompatible changes (of which I think this would be one), so I'm not holding out hope to see it any time soon.